samcli/commands/local/cli_common/invoke_context.py

```diff
@@ -300,6 +313,25 @@ def _setup_log_file(log_file):
 
         return open(log_file, 'wb')
 
+    @staticmethod
+    def _get_container_manager(docker_network, skip_pull_image):
```

Comment:
I've chosen that name, because of the surrounding code. It has `_get_{something}` pattern. Like `_get_debug_context()` which also creates a new instance